cmd/one_off_stop.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/Scalingo/cli/appdetect"
+	"github.com/Scalingo/cli/apps"
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/urfave/cli"
+)
+
+var (
+	oneOffStopCommand = cli.Command{
+		Name:     "one-off-stop",
+		Category: "App Management",
+		Usage:    "Stop a running one-off container",
+		Flags:    []cli.Flag{appFlag},
+		Description: `Stop a running one-off container
+	Example
+	  'scalingo --app my-app one-off-stop one-off-1234'
+	  'scalingo --app my-app one-off-stop 1234'`,
+		Action: func(c *cli.Context) {
+			currentApp := appdetect.CurrentApp(c)
+			if len(c.Args()) != 1 {
+				cli.ShowCommandHelp(c, "one-off-stop")
+				return
+			}
+			oneOffLabel := c.Args()[0]
+
+			// If oneOffLabel only contains digits, the client typed something like:
+			//   scalingo one-off-stop 1234
+			isNotDigit := func(c rune) bool { return c < '0' || c > '9' }
+			if strings.IndexFunc(oneOffLabel, isNotDigit) == -1 {
```

Comment:
I would say this is more readable:

```go
if ok, _ := regexp.MatchString("^[0-9]+$", oneOffLabel) ; ok {
  oneOffLabel = "one-off-" + oneOffLabel
}
```

What do you think?

Here the `isNOT`, `==`, `-1`, it makes the condition really tricky